authors/apps/articles/serializers.py

```diff
@@ -48,12 +52,28 @@ class Meta:
             'description',
             'body',
             'slug',
+            'avg_rating',
             'image_url',
             'author',
             'created_at',
             'updated_at'
         )
 
+    def get_average_rating(self, instance):
+        avg_rating = ArticleRatings.objects.filter(article=instance).aggregate(
```

Comment:
avg_rating is not descriptive enough. Consider using average_rating

Reply:
Fixed